### DISCUSSION TITLE
provider group with EIN for a hospital but without a type 2 NPI code

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#554] [General]
In a Cigna Health in_network_file (https://d25kgz5rikkq4n.cloudfront.net/cost_transparency/mrf/in-network-rates/reporting_month=2022-07/2022-07-01_cigna-health-life-insurance-company_phoenix-az-connect_in-network-rates.json.gz), I am seeing a "providers object" with a **tin** that contains an EIN for a hospital, but all of the **npi's** in the list are for individual providers (type 1) and the type 2 npi for the hospital is not in the list.

When I checked the documentation for the **npi** field in the In_Network File Schema, I found this description that seems to fairly clearly say that if the contract is with an organization, then the type 2 npi for that organization should be in the npi array:

> An array of National Provider Identifiers (NPIs). The NPI array attribute can contain a mix of Type 1 and Type 2 NPIs, both of which must be provided, if available. In contractual arrangements with Type 2 NPIs where Type 1 NPIs are unknown or otherwise unavailable, only the Type 2 NPIs must be reported.

Am I interpreting the rule correctly?

Are other people seeing this same issue in this file or other files?

Here is one example of the JSON in question, but the same issue appears is in every providers object for a hospital in this file. As a matter of fact, this file doesn't appear to contain any type 2 **npis**:

```
{
  "negotiation_arrangement": "ffs",
  "name": "X-RAY EXAM OF HAND",
  "billing_code_type": "CPT",
  "billing_code_type_version": "2022",
  "billing_code": "73130",
  "description": "X-RAY EXAM OF HAND",
  "negotiated_rates": [
    {
      "provider_groups": [
        {
          "tin": {
            "type": "ein",
            "value": "900220728"
          },
          "npi": [
            1700013687,
            1134154941,
            1285612291
          ]
        }
      ],
```

**Comment by mybijay** [comment]
Hi @nmayle - have you addressed/found solution to  this? Looking forward to hearing from you.

**Comment by tylerrollinsphd** [comment]
any solution for this issue?
